runTests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+[ "$TEST_URL" == "" ] && echo "Error: cannot find TEST_URL env var." && exit 1
+
+[ "$TEST_HEATH_URL" == "" ] && TEST_HEALTH_URL="${TEST_URL}/health"
```

Comment:
Why allow them to override it? the pipeline enforces it to be health?
convention over configuration and all

Reply:
Convention over configuration is about setting sensible defaults not about preventing people from overriding them. I agree this should generally not be needed, but it might be in special cases and it's useful to have the option.